libexec/rtld-elf/aarch64/rtld_start.S

```diff
@@ -55,6 +101,10 @@ END(.rtld_start)
  * x17 = &_rtld_bind_start
  */
 ENTRY(_rtld_bind_start)
+#ifdef __CHERI_PURE_CAPABILITY__
+	/* Currently unused and unimplemented for pure-capability code */
+	nop
```

Comment:
Make it trap with the equivalent of RISC-V's `unimp` otherwise it'll fall through to the next function.